### PR TITLE
Issue 759 create popper component

### PR DESCRIPTION
**Pull request by damian-rodriguez-sociomantic**
Pr for https://github.com/sociomantic-tsunami/nessie-ui/issues/759, also in this PR we're updating the name of the `Dropdown` component which will be called `PopUp` from now on.

### CODE REVIEW

File: src/PopperWrapper/index.jsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                    document.body,
+            ) }
+        </Manager>
+    );
+};
+
+PopperWrapper.propTypes =
+{
+    /**
+     *  Dom element to attach popper
```
Comment (conor-cafferkey-sociomantic) [inline src/PopperWrapper/index.jsx]: I think this should be something like _Reference node_  (the prop accepts a React node, not a DOM element)

File: src/PopperWrapper/index.jsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+     */
+    children       : PropTypes.node,
+    /**
+     *  id of the DOM element used as container
+     */
+    container      : PropTypes.string,
+    /**
+     *  popper element
```
Comment (conor-cafferkey-sociomantic) [inline src/PopperWrapper/index.jsx]: And here maybe _Popper content node_?

File: src/PopperWrapper/index.jsx [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+PopperWrapper.defaultProps =
+{
+    children       : undefined,
+    container      : undefined,
+    popper         : undefined,
+    popperOffset   : 'M',
+    popperPosition : 'auto',
+};
+
+PopperWrapper.displayName = 'PopperTooltip';
```
Comment (conor-cafferkey-sociomantic) [inline src/PopperWrapper/index.jsx]: Should be `'PopperWrapper'`

**Comment by conor-cafferkey-sociomantic** [comment]
Tests currently failing with:
```
Cannot find module './Popup' from 'index.js'
```

**Comment by damian-rodriguez-sociomantic** [comment]
ok, test are passing now

File: src/defaults.json [hunk 4]
```diff
@@ -303,6 +299,14 @@
 		"label": "Tell me your secrets",
 		"errorMessage": "Please enter a dark secret"
 	},
+	"Popup": {
+		"children": "Popup content",
+    "padding": "M"
```
Comment (daniel-martic-sociomantic) [inline src/defaults.json]: Indentation here 😁

**Pull request merged**